pkg/render/node.go

```diff
@@ -108,6 +109,9 @@ type NodeConfiguration struct {
 	// The health port that Felix should bind to. The controller reads FelixConfiguration
 	// and sets this.
 	FelixHealthPort int
+
+	// The bindMode
```

Comment:
"The bindMode read from the default BGPConfiguration. Used to trigger rolling updates should this value change"

Reply:
ya forgot to put full comment. Will put this one suggested by you.